### PR TITLE
add(epp): prerequisites documenation for Azure monitoring

### PR DESCRIPTION
**Pull request by lchrdn**
## Description

Adding a new documentation describing how to get tokens for setting up Azure monitoring. 

This procedure aims to be part of each "Prerequisites" part of any Azure Plugin Pack documentation to ease update management. 

## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x
- [X] 21.10.x
- [X] 22.04.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+* Get the **Client ID**:
+	- Go to **Azure Active Directory**.
+	- On the left, select **App registrations** and the name of your application. 
+	- In the **Overview** panel you will have **Application (client) ID**.
+* Get the **Client secret**:
+	- Still in **Azure Active Directory > App registrations**, go to **Certificates & secrets**.
+	- Click on **+ New client secret**, set description as well as expiration date and Add.
+	- Save the **key value**, it is the **client secret** and you won't be able to retrieve it after you leave this page.
+	
+These 4 tokens will be mandatory when using host autodiscovery and Centreon Azure based plugins.
```
Comment (itoussies) [inline en/integrations/plugin-packs/tutorials/azure-credential-configuration.md:53]:
```suggestion
These 4 tokens will be mandatory when using host autodiscovery and Centreon Azure based Plugins. 
```

File: en/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	- Choose the resource group containing the resources you want to monitor. 
+	- Click on **Access Control (IAM)** and **+ Add > Add role assignment**.
+	- Search for **Monitoring Reader role** and click Next.
+	- Select your new application as a member for this role by clicking on **+ Select members**.
+	- Review and assign.
+	
+Now that the application is created, the below steps will guide you to retrieve the 4 tokens: 
+* Get the **subscription ID**:
+	- Go to **Subscriptions** in Azure portal. 
+	- Next to the name of your subscription you have the **Subscription ID**.
```
Comment (itoussies) [inline en/integrations/plugin-packs/tutorials/azure-credential-configuration.md:39]:
```suggestion
* Get the **subscription ID**:
	- Go to **Subscriptions** in Azure portal. 
	- Copy the *Subscription ID* next to the name of your subscription.
```

File: en/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+* Get the **Tenant ID**:
+	- Go to **Azure Active Directory**.
+	- In **Overview**, under the name of the directory you will see the **Tenant ID**.
```
Comment (itoussies) [inline en/integrations/plugin-packs/tutorials/azure-credential-configuration.md:42]:
```suggestion
* Get the **Tenant ID**:
	- Go to **Azure Active Directory**.
	- Go to **Overview** and copy the *Tenant ID*.
```

File: en/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	
+* Get the **Client ID**:
+	- Go to **Azure Active Directory**.
+	- On the left, select **App registrations** and the name of your application. 
+	- In the **Overview** panel you will have **Application (client) ID**.
```
Comment (itoussies) [inline en/integrations/plugin-packs/tutorials/azure-credential-configuration.md:47]:
```suggestion
* Get the **Client ID**:
	- Go to **Azure Active Directory**.
	- Go to **App registrations** in the sidebars on the left and click on your application.
	- Go to **Overview** and copy the *Application (client) ID*.
```

File: en/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+* Get the **Client secret**:
+	- Still in **Azure Active Directory > App registrations**, go to **Certificates & secrets**.
+	- Click on **+ New client secret**, set description as well as expiration date and Add.
+	- Save the **key value**, it is the **client secret** and you won't be able to retrieve it after you leave this page.
+	
+These 4 tokens will be mandatory when using host autodiscovery and Centreon Azure based plugins. 
+
+## Azure CLI set up 
+
+*These steps are not required if you can use Azure API.*
```
Comment (itoussies) [inline en/integrations/plugin-packs/tutorials/azure-credential-configuration.md:57]: Maybe we can put this sentence in a warning format
```suggestion
> These steps are not required if you can use Azure Monitor API.
```

File: en/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+```bash
+su - centreon-engine
+az login
+```
+
+The shell will output this message including an authentication code:
+
+	*To sign in, use a web browser to open the page https://microsoft.com/devicelogin*
+	*and enter the code CWT4WQZAD to authenticate.*
```
Comment (itoussies) [inline en/integrations/plugin-packs/tutorials/azure-credential-configuration.md:77]:
```suggestion
```text
To sign in, use a web browser to open the page https://microsoft.com/devicelogin
and enter the code CWT4WQZAD to authenticate.*
```
```

**Review by itoussies — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+Two methods are available to get information from Azure:
+* Azure API ('api').
+* Azure CLI ('azcli').
+
+Centreon recommends using **Azure API** instead of Azure CLI.  
+Azure CLI method is not be compatible with host autodiscovery.
+
+## Azure API set up 
+
+For Centreon to be able to request information about Azure resources through **Azure API**, 4 tokens are needed:
```
[the remark this answers is not in the document]
Comment (Sims24) [reply en/integrations/plugin-packs/tutorials/azure-credential-configuration.md:14]: double "are"

[the remark this answers is not in the document]
Comment (itoussies) [reply en/integrations/plugin-packs/tutorials/azure-credential-configuration.md:14]: Fixed

**Review by itoussies — APPROVED** [review]

**Comment by cg-tw** [comment]
Closed by https://github.com/centreon/centreon-documentation/pull/1342

**Pull request closed without merging**